skrub/_reporting/_data/templates/tabs.css

```diff
@@ -91,5 +91,4 @@ button.tab[data-has-warning]:not(:hover):not([data-is-selected]) {
 .report-bottom-border {
     width: calc(100% - 2 * var(--radius));
     border-bottom: var(--border-s) solid var(--color-border-secondary);
-    margin: var(--spacing-m) auto 0 auto;
```

Comment:
we can replace `--spacing-m` with `--spacing-s` to make it smaller but having the bottom border touch the content looks a bit weird:

![screenshot_2025-03-20T10:09:09+01:00](https://github.com/user-attachments/assets/f3b318a5-1d18-45b6-861f-51b225c048f1)

otherwise we can also just remove that border

Reply:
I see this padding needed only the statistics and associations panel. I've added it